posts/kanban_task_switching.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Calculating the cost of task switching: why we use Kanban here at MyDrive.
+author: william
+date: "2015-03-09"
+description: Some thoughts about the detrimental affects of task switching and why Kanban is so awesome for eliminating it and maximising flow.
+tags: ["kanban", "flow", "task switching", "team work"]
+categories: ["development", "programming", "continuous improvement", "kanban"]
+---
+
+The aim of this post is to suggest that task switching is cognitively expensive and that Kanban as an agile methodology
+lends itself to reducing the cost of task switching.
+
+A beginning is a very important time in a person's career. This has recently been highlighted to me in terms
```

Comment:
Refactor the first few sentences, what you are trying to say is good but it sounds a bit rambling. Less is more.